components/org.wso2.carbon.identity.oauth.dcr/src/main/java/org/wso2/carbon/identity/oauth/dcr/service/DCRMService.java

```diff
@@ -333,11 +336,12 @@ private OAuthConsumerAppDTO createOAuthApp(ApplicationRegistrationRequest regist
         oAuthConsumerApp.setApplicationName(spName);
         oAuthConsumerApp.setCallbackUrl(
                 validateAndSetCallbackURIs(registrationRequest.getRedirectUris(), registrationRequest.getGrantTypes()));
-
         String grantType = StringUtils.join(registrationRequest.getGrantTypes(), GRANT_TYPE_SEPARATOR);
         oAuthConsumerApp.setGrantTypes(grantType);
         oAuthConsumerApp.setOAuthVersion(OAUTH_VERSION);
         oAuthConsumerApp.setTokenType(registrationRequest.getTokenType());
+        oAuthConsumerApp.setBackChannelLogoutUrl(
+                validateBackchannelLogoutURI(registrationRequest.getBackchannelLogoutUri()));
```

Comment:
What happens if registration request would not have a back channel logout URI.
Ideally, it should be an optional parameter

Reply:
in that case backchannel logout url will be an empty string.